### ISSUE TITLE
LONER: LiDAR Only Neural Representations for Real-Time SLAM. (arXiv:2309.04937v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#27635]
LONER: LiDAR Only Neural Representations for Real-Time SLAM. (arXiv:2309.04937v1 [cs.RO])<br>
https://ift.tt/qfwsvdn<br>
<p>This paper proposes LONER, the first real-time LiDAR SLAM algorithm that uses
a neural implicit scene representation. Existing implicit mapping methods for
LiDAR show promising results in large-scale reconstruction, but either require
groundtruth poses or run slower than real-time. In contrast, LONER uses LiDAR
data to train an MLP to estimate a dense map in real-time, while simultaneously
estimating the trajectory of the sensor. To achieve real-time performance, this
paper proposes a novel information-theoretic loss function that accounts for
the fact that different regions of the map may be learned to varying degrees
throughout online training. % different regions of the map having varying
degrees of uncertainty during online operation. The proposed method is
evaluated qualitatively and quantitatively on two open-source datasets. This
evaluation illustrates that the proposed loss function converges faster and
leads to more accurate geometry reconstruction than other loss functions used
in depth-supervised neural implicit frameworks. Finally, this paper shows that
LONER estimates trajectories competitively with state-of-the-art LiDAR SLAM
methods, while also producing dense maps competitive with existing real-time
implicit mapping methods that use groundtruth poses.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
